google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -198,37 +145,29 @@ public void exit(int exitCode) {
 
           if (deployOutput == null
               || deployOutput.getService() == null || deployOutput.getVersion() == null) {
-            consoleLogLn(
-                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
+            loggingHandler.print(
```

Comment:
oh right. My thinking is that the console window sort of blurs the line between UI and logging output. It _can_ be the output raw logs pipped to it. But its also the main / only form of certain information messages that the user get within the IDE when performing certain actions. But you are right that the rest of the console output (coming from the common lib) will not be localized.